crypto/asn1/asn_mime.c

```diff
@@ -552,9 +553,12 @@ int SMIME_crlf_copy(BIO *in, BIO *out, int flags)
             }
         }
     }
-    (void)BIO_flush(out);
+    ret = BIO_flush(out);
```

Comment:
I dont know if it matters but there are quite a few other (void)BIO_flush calls related to cms/pkcs7